internal/commands/app/new_app_inputs.go

```diff
@@ -57,6 +57,7 @@ type newAppInputs struct {
 	DeploymentModel realm.DeploymentModel
 	Location        realm.Location
 	Environment     realm.Environment
+	Template        string
```

Comment:
would we want `resolveTemplateID` defined on `newAppInputs` instead of `createInputs`?  also, do we want template behavior for `app init`?

Reply:
> would we want resolveTemplateID defined on newAppInputs instead of createInputs?

yeah, actually it probably makes more sense there, good call 👍 

> also, do we want template behavior for app init?

truuuue, I forgot about that command, also a good call

Reply:
hmm i think `app init` may probably have been missed during scoping but it certainly does make sense to support templates for app init, so I guess this `Template` field can stay here

Since it doesn't seem strictly necessary, i think we can skip the init command template implementation for the time being and then come back to it once we get some breathing room when AB test goes out